src/providers/pronouns/UserPronouns.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <QString>
+
+namespace chatterino::pronouns {
+
+class UserPronouns
+{
+public:
+    UserPronouns() = default;
+    UserPronouns(QString);
```

Comment:
This should have a name (the same that it has in the implementation - `pronouns`).